### DISCUSSION TITLE
Strange reading in Costsummary Powerbi

### DISCUSSION THREAD

**Discussion by DeKuiper** [microsoft/finops-toolkit#617] [Q&A] [answered]
Hi All,

Thank you for creating the Finops Hubs solutions, this looks very promising.

In order for me to start using this as the source of truth I need to understand some items in the powerbi Cost summary report (Using FinOps Hubs RC 0.2.1) .

- We have an EA and make use of both RI's and Savings plans. We also have a % of negotiated contract discount. 

- In the Summary screen under the x_discountSavings column I see both positive and negative numbers, that doesn't seem right either? Shouldn't those either be positive or negative but not both?
- in the commitments screen why do my commitment savings come out to 0? The ondemand cost and effective cost are the same, which doesn't make sense?

These are just two of the items which make me think the discount and commitment calculations are not reliable. Is this the correct place to discuss this or can I read up on this somewhere? (I did read most of the content on the FinOps Hubs pages already)

thanks,

deKuiper

**Comment by flanakin** [comment, marked as answer]
In 0.2 reports, `x_DiscountSavings` represents the total cost savings from negotiated discounts and commitment discounts. Negotiated discounts (`x_NegotiatedSavings`) are determined by comparing `ListCost` and `x_OnDemandCost` and committment discounts (`x_CommittmentSavings`) are determined by comparing `x_OnDemandCost` and `EffectiveCost`. We discovered a few issues with the Cost Management data which are making it difficult to accurately track savings. We're tracking these in issue #604. 

Specifically, `ListCost` and `x_OnDemandCost` are missing in some rows and, when available, `x_OnDemandCost` is actually incorrect. I was just informed the latter issue should have been resolved but I haven't verified it yet. I will do that as part of the 0.2.1 release.

**Reply by DeKuiper** [reply]
Thank you for the answer and I'm sorry for not finding the issue first. I'm kind of new to actively participating in Github.

Tx Marc
